public/index.html

```diff
@@ -30,6 +30,7 @@
 -------------------------------------------------------------------
 */
     </script>
+    <script src="%PUBLIC_URL%/mixpanel-lite.min.js"></script>
```

Comment:
why have you chosen to add the script directly rather than installing it? 

@mike-lvov, what would you say? is it a suitable approach for us?

Reply:
@sunny-viktoryia because it connects thought the script's adding, that is written in readme - https://github.com/john-doherty/mixpanel-lite#mixpanel-lite